JK Hera/src/main/java/ayds/hera/songinfo/moredetails/model/repository/broker/Broker.kt

```diff
@@ -0,0 +1,31 @@
+package ayds.hera.songinfo.moredetails.model.repository.broker
+
+import ayds.hera.songinfo.moredetails.model.entities.Card
+
+interface Broker {
+    fun getListOfCards(artistName: String): MutableList<Card>
+}
+
+internal class BrokerImpl(
+    private val proxyNYTimes: ProxyNYTimes,
+    private val proxyWikipedia: ProxyWikipedia,
+    private val proxyLastFM: ProxyLastFM
+): Broker{
+
+    private var listOfCards : MutableList<Card> = mutableListOf()
+
+    override fun getListOfCards(artistName: String): MutableList<Card> {
+        addCard(artistName, proxyNYTimes)
+        addCard(artistName, proxyLastFM)
+        addCard(artistName, proxyWikipedia)
+        return listOfCards
+    }
+
+    private fun addCard(artistName:String, proxy:Proxy){
+        proxy.getCard(artistName)?.let{
+            listOfCards.add(it)
+        }
+    }
```

Comment:
No es muy clean, se pasan los proxy por parámetro...  Se puede simplificar


```suggestion
    override fun getListOfCards(artistName: String): MutableList<Card> =
      mutableListOf<Card>().apply {
            proxyNYTimes.getCard(artistName)?.let{ add(it) }
            proxyLastFM.getCard(artistName)?.let{ add(it) }
            proxyWikipedia.getCard(artistName)?.let{ add(it) }
        }
```